### ISSUE TITLE
Mobile Friendly Interface

### ISSUE THREAD

**Issue by DavidZong** [DavidZong/DavidZong.github.io#4]
The interface right now isn't exactly mobile friendly. In fact it looks pretty bad. I should redesign with mobile in mind.

**Comment by codelucas** [comment]
You can try out something like a https://getbootstrap.com/ template. They are designed with mobile first in mind

**Comment by DavidZong** [comment]
Right. I'll look into it. I tried bootstrap in the past when I was doing straight up HTML and CSS editing. I'll have to read about how it plays with Jekyll. I'm assuming I just edit the templates to have bootstrap.

**Comment by DavidZong** [comment]
also how do i test on mobile without deploying?

**Comment by codelucas** [comment]
Few ways:
- after you make your local change and rebuild the site you can view the site on your browser by requesting mobile. There are also free mobile simulators you can use
- you can deploy your change to a **custom url** that is hidden from the public so only you know and can visit that url from your phone to test while debugging. After it’s ready to ship you can deploy the change to the real url.

I can show you how to do this later but it’s possible in Jekyll

- Lucas

________________________________

**Comment by DavidZong** [comment]
the chrome dev tool works pretty well.
